### PR TITLE
🌐 Add Azerbaijani translation for resources page and its subpages

### PR DESCRIPTION
**Pull request by vusallyv**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit c12a7ebc5057db9e67ed0e88e6486ed93278ae69 at: https://33b54ba2.fastapitiangolo.pages.dev

**Review by leylaeminova — APPROVED** [review]

**Review by TalehIlqar — APPROVED** [review]

**Comment by codespearhead** [comment]
Do you guys happen to know one more Azerbaijani developer you can ask a [PR review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from?

**Review by ibrahimshikhaliyev — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5a15cbba87c7fbf6e3926fe57d1f52f51296c7f2 at: https://a5896171.fastapitiangolo.pages.dev

**Comment by codespearhead** [comment]
There are 3 reviews @alejsdev . Seems to be ready to merge.

**Comment by tiangolo** [comment]
Thanks @vusallyv! :nerd_face: 

Please separate this PR into multiple PRs, one per file. That way it's easier to review, approve, and merge them.

For example, the original file for `docs/az/docs/project-generation.md` was changed, and it's now obsolete in this PR, so I can't merge the other changes with it. :sweat_smile:

File: docs/az/docs/external-links.md
Comment (nemoralis) [inline docs/az/docs/external-links.md:3]: suffix should be "-nın"

**Review by nemoralis — CHANGES_REQUESTED** [review]
There are also grammatical mistakes that I have not mentioned. This should be fixed before the merge

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**
